lib/ridgepole/delta.rb

```diff
@@ -292,6 +292,19 @@ def append_change_table_options(table_name, table_options, buf)
       buf.puts
     end
 
+    def append_change_table_raw_options(table_name, raw_table_options, table_charset, table_collation, buf)
+      if raw_table_options.blank? && ActiveRecord.gem_version >= Gem::Version.new('6.1.0')
+        # Implicit engine is InnoDB in 6.1.0
+        # related: https://github.com/rails/rails/pull/39365/files#diff-868f1dccfcbed26a288bf9f3fd8a39c863a4413ab0075e12b6805d9798f556d1R441
+        raw_table_options = 'ENGINE=InnoDB'.dup
```

Comment:
cf. https://docs.ruby-lang.org/en/2.4.0/String.html#method-i-2B-40
```suggestion
        raw_table_options = +'ENGINE=InnoDB'
```